test/end-to-end/wdio.conf.js

```diff
@@ -145,13 +145,6 @@ exports.config = {
     timeout: 90000,
   },
 
-  // always start with a clean browser session before
-  // every single test
-  beforeTest(test) {
-    browser.reload();
-    browser.windowHandleFullscreen();
-  },
```

Comment:
Why did you take this out? It still seems like a good idea to isolate tests from each other?

Reply:
Oh, sorry @martinpitt, this's for debugging! I'll add them back later. Thanks.